explorer/migrations/0009_auto_20220830_1545.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
```

Comment:
Could you remove these first two lines please? So it starts with the django imports.